Unsupported arguments for ``axis()``
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Passing more than one positional argument or unsupported keyword arguments to
`~.Axes.axis()` now raises a ``TypeError`` (such arguments used to be silently
ignored).

Length of parameter *where* in ``fill_between()`` and ``fill_betweenx()``
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
The parameter *where* in `~.Axes.fill_between()` and `~.Axes.fill_betweenx()`
must have the same length as the data points.
